packages/edit-post/src/test/editor.native.js

```diff
@@ -1,13 +1,16 @@
 /**
  * External dependencies
  */
-import renderer from 'react-test-renderer';
 import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
-
+import { mount } from 'enzyme';
```

Comment:
FYI, we're trying to move away from "enzyme" in the Gutenberg code base for several reasons, the main one being the fact that it doesn't seem well maintained anymore and updates we needed (hooks, new context...) took a long time.

React official testing tools seem to be capable of addressing most of the use-cases.

Reply:
We are using `enzyme` quite a lot. It would be nice to know a time-frame for us to also move away from it over time.

Reply:
There's no time-frame and you don't have to follow our process here I think. We think it might hurt us in the future though to keep betting on enzyme.

If I had to suggest something, I'd recommend continuing with enzyme for existing tests but avoiding it for new ones. and maybe refactor existing ones when you need to touch them for a big component rewrite or things like that.